MapboxNavigation/CarPlayManager.swift

```diff
@@ -457,48 +434,54 @@ extension CarPlayManager: CPMapTemplateDelegate {
+            overviewButton.image = UIImage(named: "carplay_overview", in: .mapboxNavigation, compatibleWith: nil)
+            
+            mapTemplate.mapButtons = [overviewButton, showFeedbackButton]
         }
-        overviewButton.image = UIImage(named: "carplay_overview", in: .mapboxNavigation, compatibleWith: nil)
-
-        mapTemplate.mapButtons = [overviewButton, showFeedbackButton]
 
         if let rootViewController = self.carWindow?.rootViewController as? CarPlayMapViewController,
             let leadingButtons = delegate?.carPlayManager?(self, leadingNavigationBarButtonsCompatibleWith: rootViewController.traitCollection, in: mapTemplate, for: .navigating) {
             mapTemplate.leadingNavigationBarButtons = leadingButtons
+        } else {
+            let muteTitle = NSLocalizedString("CARPLAY_MUTE", bundle: .mapboxNavigation, value: "Mute", comment: "Title for mute button")
+            let unmuteTitle = NSLocalizedString("CARPLAY_UNMUTE", bundle: .mapboxNavigation, value: "Unmute", comment: "Title for unmute button")
+            
+            let muteButton = CPBarButton(type: .text) { (button: CPBarButton) in
+                NavigationSettings.shared.voiceMuted = !NavigationSettings.shared.voiceMuted
+                button.title = NavigationSettings.shared.voiceMuted ? unmuteTitle : muteTitle
+            }
+            muteButton.title = NavigationSettings.shared.voiceMuted ? unmuteTitle : muteTitle
+            mapTemplate.leadingNavigationBarButtons.insert(muteButton, at: 0)
         }
         
-        let muteTitle = NSLocalizedString("CARPLAY_MUTE", bundle: .mapboxNavigation, value: "Mute", comment: "Title for mute button")
-        let unmuteTitle = NSLocalizedString("CARPLAY_UNMUTE", bundle: .mapboxNavigation, value: "Unmute", comment: "Title for unmute button")
-        
-        let muteButton = CPBarButton(type: .text) { (button: CPBarButton) in
-            NavigationSettings.shared.voiceMuted = !NavigationSettings.shared.voiceMuted
-            button.title = NavigationSettings.shared.voiceMuted ? unmuteTitle : muteTitle
-        }
-        muteButton.title = NavigationSettings.shared.voiceMuted ? unmuteTitle : muteTitle
-        mapTemplate.leadingNavigationBarButtons.insert(muteButton, at: 0)
-
         if let rootViewController = self.carWindow?.rootViewController as? CarPlayMapViewController,
             let trailingButtons = delegate?.carPlayManager?(self, trailingNavigationBarButtonsCompatibleWith: rootViewController.traitCollection, in: mapTemplate, for: .navigating) {
             mapTemplate.trailingNavigationBarButtons = trailingButtons
+        } else {
+            let exitButton = CPBarButton(type: .text) { [weak self] (button: CPBarButton) in
+                self?.currentNavigator?.exitNavigation(byCanceling: true)
+            }
+            exitButton.title = NSLocalizedString("CARPLAY_END", bundle: .mapboxNavigation, value: "End", comment: "Title for end navigation button")
```

Comment:
What if the developer wants to return an exit button in their `CarPlayManagerDelegate.carPlayManager(_:trailingNavigationBarButtonsCompatibleWith:in:for:)` implementation? There should be a method either on CarPlayManager or CarPlayNavigationViewController that creates and returns such a button, so that the developer doesn’t necessarily have to supply their own icon or localized text.